libs/service-portal/core/src/lib/navigation/masterNavigation.ts

```diff
@@ -73,6 +73,35 @@ export const servicePortalMasterNavigation: ServicePortalNavigationItem[] = [
           icon: 'people',
         },
       },
+
+      // Menntun
```

Comment:
Is this comment helpful?

Reply:
That's up for debate, it's consistent to all the other defined navigation items so it might help you skim through the list and if you are used to collapse code you are not working with and collapse the 4. level you will have a nice overview of the available navigation items